Gemfile

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+source "https://rubygems.org"
+
+# gem "rails"
+gem "rspec", "~>3.0"
```

Comment:
@iraline configura seu editor para adicionar EOF(End of file) as vezes isso pode causar problemas